components/announcement_bar/index.js

```diff
@@ -3,14 +3,14 @@
 
 import {connect} from 'react-redux';
 import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
-import {haveISystemPerm} from 'mattermost-redux/selectors/entities/roles';
+import {haveISystemPermission} from 'mattermost-redux/selectors/entities/roles';
 import {Permissions} from 'mattermost-redux/constants';
 import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
 
 import AnnouncementBar from './announcement_bar.jsx';
 
 function mapStateToProps(state) {
-    const canViewSystemErrors = haveISystemPerm(state, {perm: Permissions.MANAGE_SYSTEM});
+    const canViewSystemErrors = haveISystemPermission(state, {perm: Permissions.MANAGE_SYSTEM});
```

Comment:
Is it intentional that you are still calling the parameter `perm` here?

Reply:
Nop, I was replacing with sed, and didn't realize that perm variable. I going to change it too.